packages/salesforcedx-vscode-apex-replay-debugger/src/breakpoints/checkpointService.ts

```diff
@@ -970,10 +974,10 @@ export async function sfdxToggleCheckpoint() {
   }
   const bpAdd: vscode.Breakpoint[] = [];
   const bpRemove: vscode.Breakpoint[] = [];
-  const uri = exports.fetchActiveEditorUri();
-  const lineNumber = exports.fetchActiveSelectionLineNumber();
```

Comment:
this madness caused issues with bundling. I originally just moved to stand alone exports and ended up moving to a grouped export to work around the testing issues with copy vs reference.